### DISCUSSION TITLE
Welcome to acs-aem-commons Discussions!

### DISCUSSION THREAD

**Discussion by davidjgonzalez** [Adobe-Consulting-Services/acs-aem-commons#3298] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place for members of this community connect. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jimpres** [comment]
Hello David.  I manage the administration of AEM Assets at Nordstrom.  Recently migrated from Managed Services into Adobe Cloud Services.  Currently integrating it with WorkFront.
